vhpidirect/arrays/logicvector/run.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
+
+cd "$(dirname $0)"
+
+set -e
+
+
```

Comment:
Do we want to keep the double space here, when there are no multiple "blocks" below?

Reply:
Admittedly felt a bit silly...